pyproject.toml

```diff
@@ -93,3 +96,7 @@ section-order = [
     "numpy",
     "local-folder",
 ]
+
+[tool.jupytext.formats]
+"docs/examples_ipynb/" = "ipynb"
+"docs/examples/" = "py:light"
```

Comment:
There is the `mkdocs-jupyter` plugin already in mkdocs.yml. That was running the `finch_example.py`. 
It will try to run the new ones as well, right? And then, it will be run with jupytext too?

Reply:
Were did the `finch_example.py` go? It's not shown on the documentation website.

Reply:
I replaced it with more practical examplles.